android/src/main/java/com/airbnb/android/react/maps/AirMapView.java

```diff
@@ -578,13 +578,16 @@ public boolean dispatchTouchEvent(MotionEvent ev) {
 
         switch (action) {
             case (MotionEvent.ACTION_DOWN):
-                this.getParent().requestDisallowInterceptTouchEvent(true);
+                if (map != null && map.getUiSettings().isScrollGesturesEnabled()) {
+                    this.getParent().requestDisallowInterceptTouchEvent(true);
```

Comment:
So you're saying before this change, Android and iOS had different behavior?

If I look at the code for "Static Map", I see <MapView scrollEnabled={false} ... >. By that logic, we shouldn't scroll inside the map at all, and shouldn't capture those drags, but should instead let them scroll the ScrollView.

Which is what you describe as happening on iOS, and on Android after my change is applied. So I'm confused what the problem is?